### ISSUE TITLE
Site URL for Connecting to MetaCI

### ISSUE THREAD

**Issue by addys888** [SFDO-Tooling/MetaCI#328]
We are done with all cumulusCI configuration set up, Now we are using MetaCI CLI v0.1.4 to run these same builds against your project automatically on Heroku. We are successfully deploying a new MetaCI site on Heroku using free dyno resources.
Now we are stuck in "Adding the Repository to MetaCI" section during metaCI implementation and we have the following execution with an exception:

Traceback (most recent call last):
  File "/home/adarsh/Desktop/test-dir/venv/bin/metaci", line 11, in <module>
    sys.exit(main())
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/click/core.py", line 697, in main								
    rv = self.invoke(ctx)
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/metaci_cli/cli/config.py", line 150, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/metaci_cli/cli/commands/repo.py", line 126, in repo_list
    res = api_client('repos', 'list', params=params)
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/metaci_cli/metaci_api.py", line 27, in __call__
    resp = self.client.action(self.document, args, **kwargs)
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/coreapi/client.py", line 163, in action
    link, link_ancestors = _lookup_link(document, keys)
  File "/home/adarsh/Desktop/test-dir/venv/local/lib/python2.7/site-packages/coreapi/client.py", line 38, in _lookup_link
    raise exceptions.LinkLookupError(msg % (index_string, repr(key).strip('u')))
coreapi.exceptions.LinkLookupError: Index ['repos']['list'] did not reference a link. Key 'repos' was not found.

![image](https://user-images.githubusercontent.com/25403058/49083290-2bcf4e80-f272-11e8-84a9-e53141133347.png)

Note: We are using 'metaci repo add' or 'metaci repo list' command to adding the repository to metaCI.

**Comment by davisagli** [comment]
Hello. I tried to reproduce this issue, but actually I ran into trouble even before this step while trying to install metaci-cli. It looks like we need to work on some updates to make it work with the latest releases of CumulusCI.

One thing you can check is to make sure that your MetaCI app got successfully registered in the CumulusCI keychain. `cci service info metaci` should successfully list the URL of your heroku app along with some other details.

**Comment by addys888** [comment]
Hi,
Thanks for your response. We are using Cumulus v2.0.8 and MetaCI v0.1.4 while implementing this. we have successfully added a site on MetaCI and when we trying to add a repository to MetaCI we got the above exception. 
I also have used `cci service info metaci` command to make sure that MetaCI app got successfully registered in CumulusCI keychain.
Please find the attached image for MetaCI app responce:
![cci service info metaci_updated](https://user-images.githubusercontent.com/25403058/49155094-13c60080-f340-11e8-9dc7-1d3c02ed878e.png)

**Comment by davisagli** [comment]
https://cumulusci-test1.herokuapp.com/ appears to be serving a test page rather than MetaCI.

**Comment by addys888** [comment]
We are using a Heroku app URL while adding a site on MetaCI. We are creating two heroku app for MetaCI site, one is on Node.js and another is on PHP based app. We pass these app URL as site URL while site configuration. We faced up same exception for both the MetaCI site.

**Comment by addys888** [comment]
Thank you for your beneficial respose,
We just want to automate our builds using MetaCI. Since now we are successfully deployed MetaCI (A python based App) on Heroku platform, but while adding a repo to MetaCI using `metaci repo add` command it gives us the following exception:

![image](https://user-images.githubusercontent.com/25403058/49231504-14ca6100-f418-11e8-823b-2a702cedd930.png)

**Comment by davisagli** [comment]
@jlantz it looks like MetaCI-CLI needs to be updated to match the changes we made to repository access control in MetaCI.

**Comment by addys888** [comment]
We have made some changes over repo.py configuration file, after that we able to add a 'Repo' to MetaCI and also show the listing of repo by using `metaci repo list` command.
But still the repo we are created using `metaci repo add` command is not reflecting over [https://metci.herokuapp.com/repos/](url) MetaCI app URL. And also we got an error while executing the further commands i.e: `metaci plan list` or `metaci service list`.
